tpm2/src/tpm/formatting_tools.c

```diff
@@ -829,27 +897,31 @@ int get_ski_block_bytes(char **contents,
                         uint8_t ** block, size_t *blocksize, char *delim,
                         char *next_delim)
 {
+  // TODO: Consider passing in the delim lengths as arguments and use strnlen
```

Comment:
I am in favor of passing in the delim lengths to get_ski_block_bytes

Reply:
Ok, I'll make that change.